### ISSUE TITLE
idea?

### ISSUE THREAD

**Issue by cathblatter** [cathblatter/starfish#1]
`sf_pal_discrete <- c("#ac92a6", "#8f6593", "#006992", "#8db38b", "#04724d")`

**Comment by cathblatter** [comment]
`sf_pal_discrete2 <- c("Hunyadi yellow"="#f6ae2d",
"Orange (Pantone)" = "#f26419",
"Cerulean"="#006992",
"Cambridge blue"="#8db38b",
"Dark spring green"="#04724d")`

**Comment by cathblatter** [comment]
```
sf_pal_discrete2 <- c("Cerulean"="#006992", 
                      "Bice blue" = "#2072AF",
                      "Cambridge blue"="#8db38b", 
                      "Dark spring green"="#04724d",
                      "Hunyadi yellow"="#f6ae2d", 
                      "Orange (Pantone)" = "#f26419", 
                      "Cornell red"="#B31B1B")
```

**Comment by cathblatter** [comment]
```
sf_pal_discrete2 <- c("Cerulean"="#006992", 
                      "Cambridge blue"="#8db38b", 
                      "Dark spring green"="#04724d",
                      "Hunyadi yellow"="#f6ae2d", 
                      "Orange (Pantone)" = "#f26419", 
                      "Cornell red"="#B31B1B")



five_colors <- 
  c("Charcoal"="#264653",
  "Persian green"="#2a9d8f",
  "Saffron"="#e9c46a",
  "Sandy brown"="#f4a261",
  "Burnt sienna"="#e76f51")


ten_colors <- 
c("Imperial red"="#f94144",
  "Orange (Crayola)"="#f3722c",
  "Carrot orange"="#f8961e",
  "Coral"="#f9844a",
  "Saffron"="#f9c74f",
  "Pistachio"="#90be6d",
  "Zomp"="#43aa8b",
  "Dark cyan"="#4d908e",
  "Payne's gray"="#577590",
  "Cerulean"="#277da1")


seven_diverging <- 
c("Sky blue"="#8ecae6",
  "Blue Green"="#219ebc",
  "Cerulean"="#126782",
  "Prussian blue"="#023047",
  "Selective yellow"="#ffb703",
  "Orange peel"="#fd9e02",
  "UT orange"="#fb8500")




```

**Comment by cathblatter** [comment]
rename functions to scale_color_starfish_c / scale_color_starfish_d / scale_color_starfish_div ...

**Comment by cathblatter** [comment]
fixed by [0701720](https://github.com/cathblatter/starfish/commit/07017201de8121b072b6e85a39ddf4e2a620ecab)
